### ISSUE TITLE
wasm: shadows are broken-looking in lighting example

### ISSUE THREAD

**Issue by rparrett** [bevyengine/bevy#7397]
## Bevy version

main

## Relevant system information

chrome
```
AdapterInfo { name: "ANGLE (Apple, Apple M1 Max, OpenGL 4.1)", vendor: 4203, device: 0, device_type: IntegratedGpu, driver: "", driver_info: "", backend: Gl }
```

## What you did

with wasm-server-runner
`cargo run --example lighting --target=wasm32-unknown-unknown`

## What went wrong

![image](https://user-images.githubusercontent.com/200550/215272892-1ed537f6-8878-4884-8481-eb152fc4bf06.png)

## Additional information

The shadow cascades are adjusted for this example, but in a way that breaks them on wasm due to `num_cascades` being > 1.

`load_gltf` may be broken too, but if it is, it's less obvious.

Both of these examples are built for https://bevyengine.org/examples/

related: #7362

**Comment by danchia** [comment]
Ah yes, I always forget about wasm :( The cascades default settings account for it, but not the examples

**Comment by danchia** [comment]
Maybe the right thing to do for WASM is to also do the clamping in the cascades config constructor - that way we'll at least we can use the far bound.

**Comment by rparrett** [comment]
Fixed by #7456
